### PR TITLE
🔧 various api fixes

### LINKED ISSUE #15: Boolean name for `content.reverse` in layout styles
`isReversed`?

### LINKED ISSUE #16: Rename `content.size` to `content.dimensions` in layout styles?

### LINKED ISSUE #17: Move isSelectable to text styles
Since it does not really make much sense in layout.

### LINKED ISSUE #18: Rename `inset` to `isInset` in shadow, since it's a boolean

### LINKED ISSUE #21: `box-sizing: border-box` by default for everything?
Both Layout and Text?

@xaviervia

### PR DESCRIPTION
**Pull request by xaviervia**
Closes #18
Closes #17
Closes #21
Closes #16
Closes #15

### CODE REVIEW

**Comment by deepsweet** [comment]
don't forget to `yarn start testWatch` and fix it :)

**Comment by xaviervia** [comment]
I’m going to add all the minor updates in bulk, then fix snapshots in one pass :)

**Comment by xaviervia** [comment]
Hahaha, nah, just lazy.

**Comment by deepsweet** [comment]
https://www.youtube.com/watch?v=d0RJDsH1D8g

**Comment by xaviervia** [comment]
Ready now

File: packages/validate/src/schemas/layout.js [hunk 1]
```diff
@@ -80,9 +80,9 @@ export default {
         shouldWrap: { type: 'boolean' }
       }
     },
-    cursor: { type: 'string' },
+    cursor: { enum: ['string'] },
```
Comment (deepsweet) [inline packages/validate/src/schemas/layout.js]: nice, didn't know that it works like this :)

Comment (deepsweet) [reply packages/validate/src/schemas/layout.js]: but wait, how does it know that it's not a single `s"tring"` enum?

**Comment by deepsweet** [comment]
Hold on, fixing it.

**Pull request merged**